src/svg/counter.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 32 32"><path fill="#000" d="M14.858 5.91l1.59.29a11.36 11.36 0 019 8.5l.47 2-.22 2a11.37 11.37 0 11-12.42-12.64l1.58-.15zm.14-5.91c-1.66 0-3 .45-3 1 0 .29.39.56 1 .74v1.33A14.37 14.37 0 1028.648 19h2c.75 0 1.35-1.12 1.35-2.5s-.6-2.5-1.33-2.5h-2.33a14.38 14.38 0 00-11.34-10.75V1.74c.61-.18 1-.45 1-.74 0-.55-1.34-1-3-1z"/><path fill="#000" d="M6.918 15.35v-.56c1.06 0 1.3-.16 1.3-.88h.69v6.21h-.71v-4.77h-1.28zM10.608 20c0-1.67 3.23-2.56 3.23-4.39a1 1 0 00-1.14-1.14c-.92 0-1.3.66-1.3 1.52h-.72a2 2 0 012.07-2.17 1.64 1.64 0 011.83 1.79c0 1.81-2.55 2.88-3.06 3.84h3.48v.66h-4.39V20zM16.938 14.91h.71c0 .82.58 1.3 1.49 1.3s1.47-.44 1.47-1.21-.76-1.19-1.57-1.19h-.32v-.61h.33c.71 0 1.25-.27 1.25-1s-.47-1-1.17-1a1.11 1.11 0 00-1.22 1.12h-.69a1.76 1.76 0 011.91-1.77 1.68 1.68 0 011.87 1.55 1.36 1.36 0 01-.9 1.32 1.52 1.52 0 011.24 1.58c0 1.13-.88 1.85-2.22 1.85a1.94 1.94 0 01-2.18-1.94zM19.828 22.31h-3.06v-.44l3.12-4.29h.64v4.07h1v.66h-1v1.48h-.7v-1.48zm0-.66v-3l-2.13 3h2.13z"/></svg>
```

Comment:
Some cleanup needed.

- `viewBox` should be 24x24, not 32x32
- `fill="none"` in the `<svg>` tag should be removed
- for the 2 `<path>` the `fill` should be `"currentColor"` rather than a define color like `#000` (this way, we have better control over the color with CSS)